__tests__/app.test.js

```diff
@@ -49,4 +49,73 @@ describe('GET api', () => {
     })
 })
             
-            
\ No newline at end of file
+ describe('GET /api/articles', () => {
+    test('GET 200: should return an array of objects containing all articles', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            const articles = body.articles;
+            expect(articles).toHaveLength(13);
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    article_id: expect.any(Number),
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    comment_count: expect.any(Number)
+                })
+            })
```

Comment:
Good job with this one too, but you should also check the order. jest-sorted would be my advice!